packages/node_modules/samples/browser-single-party-call-with-mute/app.js

```diff
@@ -303,7 +303,7 @@ document.getElementById('dialer').addEventListener('submit', (event) => {
         return joinMeeting(meeting);
       });
     })
-    .catch((error) => {
+    .catch(({error}) => {
```

Comment:
What is causing the error object to have its own error object?

Reply:
It seems that it wraps an error in a MediaError, which it self is an extension of the Error object

```js
 new MediaError(new Error({
	name: err.name,
    localMediaFailed: true,
    cameraFailed: err === mediaConfig.permissionDenied
}), 'Local Get User Media failed')
```

https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/@webex/plugin-meetings/src/media/index.js#L339

https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/@webex/plugin-meetings/src/common/errors/media.js

Reply:
![img](https://media1.giphy.com/media/l3q2K5jinAlChoCLS/giphy.gif)

Reply:
We need to fix the code for the error generation and not the displaying of the error. Users shouldn't need to know to deconstruct multiple levels of errors to get an error message.